docs/superpowers/plans/2026-04-29-self-ref-chain-fixture-test.md

```
@@ -0,0 +1,320 @@
+  (the closing brace of `debug_real_worldwide_data_r`, followed by a blank line before the next item in `mod tests`).
+
+  The easiest way: search for `fn debug_real_worldwide_data_r` to find the exact current line, then delete from the `///` comment above `#[test]` down through the closing `}` of the function.
+
+- [ ] **Step 2: Verify the test suite still compiles and the new test still passes**
+
+  ```bash
+  cargo test -p raven self_ref_variable_not_in_scope_across_deep_source_chain 2>&1 | tail -10
+  ```
+
+  Expected:
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)